templates/veoibd.yaml

```diff
@@ -83,30 +98,28 @@ Resources:
         Type: EC2
         MaxvCpus: 256
         SecurityGroupIds:
-          - !ImportValue us-east-1-computevpc-VpnSecurityGroup
+          - Fn::ImportValue:
+              !Sub '${AWS::Region}-${VpcName}-VpnSecurityGroup'
         Subnets:
-          - !ImportValue us-east-1-computevpc-PrivateSubnet
-          - !ImportValue us-east-1-computevpc-PrivateSubnet1
-          - !ImportValue us-east-1-computevpc-PrivateSubnet2
+          - Fn::ImportValue:
+              !Sub '${AWS::Region}-${VpcName}-PrivateSubnet1'
+          - Fn::ImportValue:
+              !Sub '${AWS::Region}-${VpcName}-PrivateSubnet2'
         MinvCpus: 0
         ImageId: !Ref ComputeImageId
         InstanceRole: !GetAtt BatchInstanceProfile.Arn
         InstanceTypes:
           - optimal
         DesiredvCpus: 8
         Tags:
-          "Name": !Sub '${AWS::StackName}-BatchComputeEnvironment'
+          "Name": !Sub '${AWS::Region}-${AWS::StackName}-BatchComputeEnvironment'
           "Department": !Ref Department
           "Project": !Ref Project
           "OwnerEmail": !Ref OwnerEmail
   BatchJobQueue:
     Type: "AWS::Batch::JobQueue"
     Properties:
-      JobQueueName: !Join
-        - '-'
-        - - !Ref AWS::StackName
-          - "job"
-          - "queue"
+      JobQueueName: !Sub '${AWS::Region}-${AWS::StackName}-BatchJobQueue'
```

Comment:
you want this to be `!Sub '${AWS::Region}-${AWS::StackName}-BatchJobQueue'`

Reply:
The BatchJobQueue name is not required either. Should I remove that?